.gitmodules

```diff
@@ -63,3 +63,7 @@
 	path = thirdparty/json/nlohmann_json
 	url = https://github.com/nlohmann/json.git
 	shallow = true
+[submodule "thirdparty/flatbuffers/flatbuffers"]
+	path = thirdparty/flatbuffers/flatbuffers
+	url = https://github.com/ilya-lavrenov/flatbuffers.git
```

Comment:
Shouldn't it link to official repository?

Reply:
It will, once https://github.com/google/flatbuffers/pull/7801 is merged

Reply:
@t-jankowski please, see https://github.com/openvinotoolkit/openvino/pull/15389